README.md

```diff
@@ -11,8 +11,26 @@ organisation of neuroimaging data.
 In this repository, we develop the
 [BIDS specification](https://bids-specification.readthedocs.io/en/latest/).
 
-**Want to learn more about working with BIDS? Have a question, comment, or suggestion?**
-Open or comment on one of our [NeuroStars issues](https://neurostars.org/tags/bids) or check out the [BIDS Starter Kit](https://github.com/bids-standard/bids-starter-kit)!
+# Using BIDS
```

Comment:
"Using BIDS" here means curating your own dataset to conform to BIDS. I think most people would think about "using" to mean running something. What about:

```suggestion
# Applying BIDS to your data
```

Or just

```suggestion
# Applying BIDS
```